nss-day-cohort-47/reactive-nutshell-captain-america-onliners
# cc-morecssandlogoutbutton

# Description
A lot of the styling was changed, and a functional logout button was added to the navbar
Fixes # (issue)
Cosmetic and functional
Please delete options that are not relevant.
# Testing Instructions
Use your eyes to observe the changes and click the logout button to make sure it logs you out
# Checklist:
- [x] I have added test instructions that prove my fix is effective or that m
